service/src/test/java/com/fitgoal/service/impl/NotificationServiceImplTest.java

```diff
@@ -0,0 +1,66 @@
+package com.fitgoal.service.impl;
+
+import com.fitgoal.api.NotificationService;
+import com.fitgoal.api.domain.Recipient;
+import com.fitgoal.api.domain.UserVerification;
+import com.fitgoal.dao.AuditDao;
+import com.fitgoal.dao.domain.AuditDto;
+import com.fitgoal.service.impl.utils.TestHelper;
+import com.fitgoal.service.mail.impl.MailSenderImpl;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentMatchers;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.mockito.Mockito.verify;
+
+@RunWith(MockitoJUnitRunner.class)
+public class NotificationServiceImplTest {
+
+    @Mock
+    private AuditDao auditDao;
+    @Mock
+    private MailSenderImpl mailSenderImpl;
+
+    private NotificationService notificationService;
```

Comment:
you can use a feature of the MockitoJUnitRunner.class -> just put the @InjectMocks annotation and you can remove the whole @Before method, mockito will create and inject all the @Mock dependencies for you

Reply:
Refactored